dev/io.openliberty.session.6.1.internal/src/io/openliberty/session/http/HttpSessionImpl61.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.session.http;
+
+import java.util.logging.Level;
+
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.session.SessionContext;
+import com.ibm.ws.session.utils.LoggingUtil;
+import com.ibm.wsspi.session.ISession;
+
+import io.openliberty.session.AbstractSessionData61;
+import jakarta.servlet.ServletContext;
+
+/**
+ * Create instance and pass the ISession, along with others contexts, into the parent's hierarchy
+ */
+public class HttpSessionImpl61 extends AbstractSessionData61 {
```

Comment:
Similarly can the AbstractSessionData61 class be removed and its content be added here?